auth/nginx/nginx.conf

```diff
@@ -0,0 +1,14 @@
+#this config says that whenever you call /api/v1 forward the request to flask:5000 using uswgi. Flask is the name of the service that we will create in our docker-compose file
```

Comment:
Erm, this comment doesn't seem to be about this file?

Reply:
I have copied this file from maybe dash or deployment

Reply:
Just remove this comment then